recipes/verta/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "verta" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 24db7f5922fd63db8094717a020560435e121f5bf9c7c9478bad5938fe08947e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Do you need a compiler? If so, this (probably) can't be a `noarch: python` package.

Reply:
I'm admittedly unclear about this. All the original code in this package is pure Python, but I wasn't sure if any of its dependencies require C. I should probably omit the compiler in this case, right?

Reply:
At some point they will, but only direct dependencies need to declare it so yes you don't need it in this case.

Reply:
Got it; removed the compiler requirement. Thanks!